gaea/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
```

Comment:
hi @andy-ms , at first I wrote a module gaea-model, I hope to share the namespace in multiple NPM packages, and now I hope to be able to install through @types/gaea-model, the usage is above, thank you!
PS: test file has been added.